### ISSUE TITLE
Doubt about Backward Function

### ISSUE THREAD

**Issue by arc130** [sniklaus/pytorch-liteflownet#11]
I think the backward function may be Implemented error.
the tensorGrid is in range [-1, 1] and tensorFlow should also be normalized in it. 
your code  
```
tensorFlow[:, 0:1, :, :] / ((tensorInput.size(3) - 1.0) / 2.0)
```
I think it should be :
```
2.0 * (tensorFlow[:, 0:1, :, :] / (tensorInput.size(3) - 1.0)) - 1.0 
```

**Comment by sniklaus** [comment]
Thank you for critically examining my code! Please note that the estimated `tensorFlow` itself can have negative values and the only reason for the division is to scale `tensorFlow` in accordance with the image size. Try your suggested version, execute the provided `run.py`, and examine the result, it will probably have little to no meaning anymore.

**Comment by legen-wait40-dary** [comment]
Hello, 
Is there any reason why do scaling with ` ((tensorInput.size(3) - 1.0) / 2.0)`
instead of `(tensorInput.size(3) - 1.0)`?
Does the line assume that the input tensorflow stay in [-(w-1)/2, (w-1)/2] so result will be [-1, 1]?
